.cicd/pipeline.yml

```diff
@@ -3,78 +3,100 @@ env:
   TEST_TIMEOUT: 60
   TIMEOUT: 120
   VERBOSE: true
+  DEBUG: true
 
 steps:
   - wait
 
   - label: ":webhook: Trigger Travis CI Build"
-    command: |
-      . ./.cicd/trigger-travis.sh
+    command:
+      - "./.cicd/travis/travis-trigger.sh"
     agents:
-      queue: "automation-eos-builder-fleet"
+      queue: "automation-basic-builder-fleet"
 
   - trigger: "eosio-base-images-beta"
-    label: ":docker: Ensure base images exist"
+    label: ":docker: Ensure Base Images"
     build:
       commit: "${BUILDKITE_COMMIT}"
       branch: "${BUILDKITE_BRANCH}"
 
   - wait
 
-  - label: ":aws: Amazon_Linux 2 - Build"
+  - label: ":aws: Amazon_Linux 2 - Build & Test"
     agents:
       queue: "automation-eos-builder-fleet"
     plugins:
       - docker#v3.2.0:
           debug: $DEBUG
-          image: "eosio/producer:eosio-amazonlinux-2-1c266e25276ad1f1147a4df5c12921968079c49b"
+          image: "eosio/producer:eosio-amazonlinux-2-cc8d19ea479663d98189ce2473aacc2a7567a8be"
+          environment:
+            - "BUILDKITE_AGENT_ACCESS_TOKEN" # Needed for buildkite-agent upload/download
+          mount-buildkite-agent: false # Mounting bk-agent doesn't work, so disable it
+          propagate-environment: true # Need for buildkite-agent upload/download (JOB_ID, etc)
     timeout: $BUILD_TIMEOUT
     skip: $SKIP_AMAZON_LINUX_2
 
-  - label: ":centos: CentOS 7.6 - Build"
+  - label: ":centos: CentOS 7 - Build & Test"
```

Comment:
This does not follow the label guidelines established [here](https://github.com/EOSIO/eos-buildkite-pipelines/pull/27).

Reply:
Should we fix the dockerfile to use FROM: centos:7.6.1810?

Reply:
Also, why isn't this a problem for amazonlinux?